tests/utils.py

```diff
@@ -864,7 +865,7 @@ def snapshot_context(token, ignores=None, tracer=None, async_mode=True, variants
         # clear queue in case traces have been generated before test case is
         # itself run
         try:
-            tracer._writer.flush_queue()
+            tracer.flush()
```

Comment:
Can this be pulled out along with the other call to the internal function as a separate PR?

https://github.com/DataDog/dd-trace-py/blob/39ae0e01ad74524729cf0fb2c0beecc478c78268/tests/utils.py#L899